### DISCUSSION TITLE
pra quem está comecando agora. O que fazer de projeto?por onde comecar?

### DISCUSSION THREAD

**Discussion by camila1467** [frontendbr/forum#1652] [Discussão]
bom. Minha duvida é bem simples. Eu to começando na área e estudando desenvolvimento, porem, vejo muitas pessoas dizendo sobre fazer projetos.. eu entro aqui e vejo muita coisa que não entendo,sei que é uma vasta área e tem muuuita coisa pra se aprender. Quando dizem "faça projetos pra iniciar" no que se referem?

**Comment by robertofilho1** [comment]
Camila, quando a galera se refere a projetos, dependendo da área de atuação, significa vc fazer pequenos desenvolvimentos. Exemplo: Para aprender Flutter eu fiz um app para controle de entrada de pessoas em eventos, onde fiz toda a parte de UX/UI, prosseguindo até no desenvolvimento Back/Front da aplicação.
Se vc não tem idéias de projetos, pode pesquisar alguns aqui no próprio Git, ou clonar uma grande pagina (a home da Nubank é muito boa para praticar).
Espero ter ajudado e bons estudos.

**Comment by ninetails** [comment]
> Quando dizem "faça projetos pra iniciar" no que se referem?

Sobre isso em específico, [recomendo o texto do @felipefialho](https://www.felipefialho.com/blog/aproximando-seus-estudos-de-cenarios-reais/), lá cita um repo muito legal dele também com propostas de desafios utilizados em testes.

**Comment by naow-dev** [comment]
Eu também estou no começo dessa jornada de frontend.
Eu fico estudando e estudando, vendo livros, vídeos, etc... e não sei aonde aplicar.
Então eu gosto muito do Linux e resolvi fazer o redesign de alguns aplicativos, criar um site mais intuitivo pra eles, afim de pegar experiência. 

Uma ideia bem bacana também agora é que segunda feira vai começar a Semana Omnistack 10. Lá você vai ter uma ajuda bem legal da onde aplicar os conhecimentos.
[https://rocketseat.com.br/week-10/inscricao](url)

**Comment by jq2** [comment]
Bem, acho que a minha resposta no outro fórum (backend-br/forum#96) pode ajudar outros membros aqui.

**Comment by felipefialho** [comment]
Oi @camila1467, tudo bem?

No finalzinho do ano passado escrevi 3 artigos que podem te ajudar a dar algum norte:

- [O que Front-end Developers precisam saber](https://www.felipefialho.com/blog/o-que-front-end-developers-precisam-saber/)
- [Como estudar, praticar e conseguir oportunidades como (Front-end) Developer](https://www.felipefialho.com/blog/como-estudar-praticar-e-conseguir-oportunidades-como-front-end-developer/)
- [Como se atualizar sobre Desenvolvimento](https://www.felipefialho.com/blog/)

A ideia é que esses artigos se completem, então tem bastante informação 😄

**Comment by caioincau** [comment]
Acho que o pessoal está querendo dizer projetos para praticar, se você não tem nenhuma base ainda, recomendo começar com algum curso ou livros mais focado em HTML, CSS e JS.

A Fernanda Bernardo fez um curso para iniciantes: https://www.udemy.com/course/frontend-basico-do-zero-a-uma-pagina-com-html-e-css/

**Comment by bbertolani** [comment]
Pra galera que gosta de back tb
https://levelup.gitconnected.com/10-great-programming-projects-to-improve-your-resume-and-learn-to-program-74b14d3e9e16
